pkg/rules/rulespb/custom.go

```diff
@@ -54,11 +53,11 @@ func NewRecordingRule(r *RecordingRule) *Rule {
 //
 // Note: This method assumes r1 and r2 are logically equal as per Rule#Compare.
 func (r1 *RecordingRule) Compare(r2 *RecordingRule) int {
-	if r1.LastEvaluation.Before(r2.LastEvaluation) {
+	if r1.LastEvaluation.Compare(r2.LastEvaluation) < 0 {
```

Comment:
@GiedriusS @squat , wdyt about this `Compare()` in replacement for go time.Time's `Before() and After()`???